src/app/calc.css

```
@@ -0,0 +1,26 @@
+#head{
```

use prettier on css files as well. Should be:

```suggestion
#head {
```

---

Check ✅